components/blitz/resources/omero/api/IAdmin.ice

```diff
@@ -17,9 +17,6 @@ module omero {
 
     module api {
 
-        /**
-         * See <a href="http://hudson.openmicroscopy.org.uk/job/OMERO/javadoc/ome/api/IAdmin.html">IAdmin.html</a>
```

Comment:
Ah sorry misunderstood you and got confused between the blitz api doc and the javadoc. I don't know how we feel in general with using latest redirects for this but `http://downloads.openmicroscopy.org/latest/omero5.1/api/ome/api/*` is certainly the most appropriate URLs and I can use these instead of removing these see blocks. That being said, this implies a 5.1.x slice2html API page will link to a 5.1.y javadoc page (arguably this is anyways better than broken links as we currently have). /cc @hflynn